include/tester/db/Database.hpp

```diff
@@ -169,17 +95,15 @@ class Database
 
 
 
-    bsoncxx::stdx::optional<bsoncxx::document::value> getLog(const std::string& collection_name, const std::string& id)
+    bsoncxx::stdx::optional<bsoncxx::document::value> getLog(std::string collectionName, std::string id)
```

Comment:
same here regarding the `static constexpr`

Reply:
done